benches/benches/bevy_tasks/iter.rs

```diff
@@ -63,11 +63,11 @@ fn bench_for_each(c: &mut Criterion) {
     let mut v = (0..10000).collect::<Vec<usize>>();
     c.bench_function("for_each_iter", |b| {
         b.iter(|| {
-            v.iter_mut().for_each(|x| {
```

Comment:
i'm suddenly realising that the point of this bench test is to bench the `for_each` function...
i'll add an explicit "allow" here